Zappy_IA_Report/DTC/Code/data.py

```
@@ -0,0 +1,62 @@
+FEATURES = [
+    "food", "linemate", "sibur", "deraumere", "mendiane",
+    "phiras", "thystame", "level", "players_nearby"
+]
+model = None
+
+def load_data() -> pd.DataFrame:
+    if os.path.exists(DATASET_FILE):
+        return pd.read_csv(DATASET_FILE)
+    return pd.DataFrame(columns=FEATURES + ["actions"])
+
+def save_data(df:pd.DataFrame) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)